enterprise/server/auth/auth.go

```diff
@@ -469,23 +484,24 @@ func newOpenIDAuthenticator(ctx context.Context, env environment.Env, oauthProvi
 		return nil, err
 	}
 
-	// Set the JWT key.
-	jwtKey = []byte(env.GetConfigurator().GetAuthJWTKey())
-
 	// Initialize API Key -> Group cache unless it's disabled by config.
 	var akgCache *apiKeyGroupCache
-	if env.GetConfigurator().GetAuthAPIKeyGroupCacheTTL() != "0" {
+	if *apiKeyGroupCacheTTL != time.Duration(0) {
```

Comment:
This is an (unintentional) change in behavior. 
The cache is on by default and disabled if the config value is set to "0" (vs the default config value of "").
Now that we have default flag values you could set the default value of apiKeyGroupCacheTTL to 5 minutes and get rid of defaultAPIKeyGroupCacheTTL